README.txt

```diff
@@ -0,0 +1,5 @@
+HURRAY
+
+
```

Comment:
Could you remove unnecessary lines spaces

Reply:
ok